Composer/packages/extensions/obiformeditor/src/Form/fields/PromptField/Validations.tsx

```diff
@@ -168,6 +161,48 @@ export const Validations: React.FC<ValidationsProps> = props => {
           />
         ))}
       </div>
+      <div css={[validationItemInput, field]}>
+        <div css={validationItemValue}>
+          <ExpressionWidget
+            id={id}
+            value={newValidation}
+            onChange={handleChange}
+            placeholder={formatMessage('Add new validation rule here')}
+            onKeyDown={submitNewValidation}
+            schema={schema}
+            formContext={formContext}
+            rawErrors={[]}
```

Comment:
Sure you want to send an empty array but not leaving it `undefined`? Then, use `const EMPTY_ARRAY = [];` outside of this function, so the reference never change and you don't need `useMemo`.